internal/convert/database_user.go

```diff
@@ -65,6 +65,21 @@ func GetStringPointerIfNotEmpty(input string) *string {
 	return nil
 }
 
+func GetArrayPointerIfNotEmpty(input []string) *[]string {
+	if len(input) > 0 {
+		return &input
+	}
+	return nil
+}
```

Comment:
there's a `pointer` package which this may be better fitted for

Reply:
Thanks for the tip, I've moved all these functions in the latest commit